model/clusters_mgmt/v1/cluster_type.model

```diff
@@ -186,4 +186,7 @@ class Cluster {
 
 	// Node drain grace period.
 	NodeDrainGracePeriod Value
+
+	// Indicates whether that etcd is encrypted or not
```

Comment:
Try to end sentences in comments with punctuation: `Indicates ... or not.`. It isn't very relevant here, or now. But in the future I would like to automatically generate AsciiDoc documentation from these comments, and the tool to do that will most likely use the dot to detect the end of the sentence and to use it as the summary of the documentation for that item.

I'd also suggest to elaborate a bit in the documentation to explain if this is something that can be set only when the cluster is initially created or if it can be updated once the cluster is installed.

Reply:
Adding the dot. For now, we do it only for creation but we might do it for patching in the future. The question is if it worth adding a comment now for cluster creation and when patching is done, to update this comment..... WDYT?

Reply:
I think it is worth adding and later updating.